src/shoobx/wfmc/process.py

```diff
@@ -1156,7 +1162,7 @@ def __repr__(self):
 @interface.implementer(interfaces.IParameterDefinition)
 class Parameter(object):
 
-    input = output = False
+    input = output = skipInputEval = False
```

Comment:
just nitpicking: I somehow don't like this style, I'd prefer one line per attribute:
```
input = False
output = False
skipInputEval = False
```